src/components/utils/Error404.tsx

```diff
@@ -0,0 +1,22 @@
+import React from 'react';
+import bgVideo from '../../assets/video/A32NX.mp4';
+
+export const Error404 = () => {
+    return (
+        <>
+            <div className="mx-auto my-80 max-w-md md:max-w-7xl text-center ">
```

Comment:
```suggestion
            <div className="relative mx-auto px-2 my-80 max-w-md md:max-w-4xl text-center z-20">
```

Reply:
This makes sure that the text is above the video and adjusts padding for mobile

Reply:
md:max-w-4xl looks a little better for desktop so it scrunches it into a box rather than single line text shown here:
![image](https://user-images.githubusercontent.com/18389085/108490298-86618480-729a-11eb-8645-423b120e24c9.png)